examples/notifications-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: kube-monkey-config-map
+      namespace: kube-system
+    data:
+      config.toml: |
+        [kubemonkey]
+        run_hour = 20
+        start_hour = 21
+        end_hour = 23
+        blacklisted_namespaces = ["kube-system"]
+        whitelisted_namespaces = [ "default", "test-kill-namespace" ]
+        time_zone = "Australia/Melbourne"
+        graceperiod_sec= 10
+        [notifications]
+          enabled = true
+          [[notifications.attacks]]
```

Comment:
I wonder if there is value in allowing multiple notifiers. I had a look at other projects and at some internal tools used at work and I could not find any that would allow more than one. Maybe that's an edge case and we can keep this simple with just one?

ie, the user either reports to slack, or to something else. but not to multiple places.